### PR TITLE
kraken: add impacts on stop_point and stop_area in display_informations.links

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket concerned: https://jira.canaltp.fr/browse/NAVITIAII-2267

### CODE REVIEW

File: source/type/pb_converter.cpp [hunk 1]
```diff
@@ -1258,8 +1258,31 @@ void PbCreator::Filler::fill_messages(const VjStopTimes* vj_stoptimes,
...
     }
+
+    // here we add all the impacts on stop_point and stop_area of section.from
+    const auto& from_sp = vj_stoptimes->stop_times.front()->stop_point;
+    for (const auto& sp_message: from_sp->get_applicable_messages(pb_creator.now, pb_creator.action_period)) {
+        fill_message(sp_message, pt_display_info);
+    }
+
+    const auto& from_sa = vj_stoptimes->stop_times.front()->stop_point->stop_area;
+    for (const auto& sp_message: from_sa->get_applicable_messages(pb_creator.now, pb_creator.action_period)) {
```
Comment (xlqian) [inline source/type/pb_converter.cpp]: minor:  ~`sp_message`~ -> `message`

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 2]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
 
         assert len(response['disruptions']) == 0
 
+    def test_disruption_on_journey_display_informaions(self):
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: typo: informa`t`ions

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 3]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
...
+        we add one disruption on stop_area of the section and we should get it in display_informations.links[]
+        we add one on line and we should also get it in display_informations.links[]
+        we add one disruption on stop_point of the section and we should get it in display_informations.links[]
+        """
+
+        #we already have created a disruption on stopA (stop_area stopB)
+        query = journey_basic_query+'&_current_datetime=20160314T144100'
+        response = self.query_region(query)
+
+        #the response must be still valid (this test the kraken data reloading)
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: this test`s`

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 4]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
...
+        self.is_valid_journey_response(response, query)
+
+        #we search for the disruption on stop_area in section.disdisplay_informations.links[]
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: typo: `dis`display

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 5]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
...
+
+        #we create a disruption on line
+        self.send_mock("bob_the_disruption_on_line", "A", "line")
+        query = journey_basic_query+'&_current_datetime=20160314T144100'
+        response = self.query_region(query)
+
+        #the response must be still valid (this test the kraken data reloading)
+        self.is_valid_journey_response(response, query)
+
+        #we search for the disruption on stop_area and line in section.disdisplay_informations.links[]
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: typo: `dis`display

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 6]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
...
+        assert len(links) == 2
+        assert any(link['id'] == 'impact_bob_the_disruption_1' for link in links)
+        assert any(link['id'] == 'impact_bob_the_disruption_on_line_1' for link in links)
+
+        #we create a disruption on a stop_point A
+        self.send_mock("bob_the_disruption_on_stop_point", "stop_point:stopA", "stop_point")
+        query = journey_basic_query+'&_current_datetime=20160314T144100'
+        response = self.query_region(query)
+
+        #the response must be still valid (this test the kraken data reloading)
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: this test`s`

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 7]
```diff
@@ -297,6 +297,64 @@ def test_journey_with_current_datetime_out_of_bounds(self):
...
+        self.is_valid_journey_response(response, query)
+
+        #the response must be still valid (this test the kraken data reloading)
```
Comment (pbougue) [inline source/jormungandr/tests/chaos_disruptions_tests.py]: this test`s`

**Review by pbougue — APPROVED** [review]
just typos

**Comment by kadhikari** [comment]
@pbougue i pushed some corrections on your comments

**Pull request merged**